migrations/v1_9_6-v1_10_0.md

```diff
@@ -216,3 +216,32 @@ ON data.aws_collect_iam_get_account_authorization_details
 TO ROLE app_snowalert
 ;
 ```
+
+## EC2 DESCRIBE SNAPSHOTS 
```

Comment:
please upper case acronyms, but title-case the titles, i.e. "EC2 Describe Snapshots"